pangeo_forge_recipes/transforms.py

```diff
@@ -446,25 +445,33 @@ def expand(self, references: beam.PCollection) -> beam.PCollection:
 @dataclass
 class WriteCombinedReference(beam.PTransform, ZarrWriterMixin):
     """Store a singleton PCollection consisting of a ``kerchunk.combine.MultiZarrToZarr`` object.
-
     :param store_name: Name for the Zarr store. It will be created with
         this name under `target_root`.
     :param target_root: Root path the Zarr store will be created inside;
         `store_name` will be appended to this prefix to create a full path.
-    :param output_json_fname: Name to give the output references file. Must end in ``.json``.
+    :param target_chunks: Dictionary mapping dimension names to chunks sizes.
+        If a dimension is a not named, the chunks will be inferred from the data.
+    :param output_file_name: Name to give the output references file.
+    :param output_file_type: Filetype to store reference files as. Options are: "parquet" or "json".
+    :param concat_dims: concat_dims kwarg to pass to write_combined_reference if using
+    .parquet as a storage format.
     """
 
     store_name: str
     target_root: Union[str, FSSpecTarget, RequiredAtRuntimeDefault] = field(
         default_factory=RequiredAtRuntimeDefault
     )
-    output_json_fname: str = "reference.json"
+    output_file_name: str = "reference"
+    output_file_type: str = "json"
```

Comment:
See question above about have a single kw and inferring the file type from the suffix.